pkg/job/manager_test.go

```diff
@@ -2,12 +2,13 @@ package job
 
 import (
 	"fmt"
+	"net"
 	"testing"
 
-	"github.com/nov1n/kubernetes-workflow/pkg/client"
-
-	"k8s.io/kubernetes/pkg/api"
+	k8sApi "k8s.io/kubernetes/pkg/api"
```

Comment:
Why do you do this?
I think we should only use this when there is a collision with our own _api_ package.